### ISSUE TITLE
Custom Classifier

### ISSUE THREAD

**Issue by baileyfriend** [anantdgoel/HackPrincetonF16#24]
Hey!
Firstly, I love your idea here.
Secondly, would you be open to creating your own classifier in python (Using Sci-Kit Learn or Tensorflow) to begin to categorize valid and invalid news sites instead of using a static list defined elsewhere? This would open the product up to being able to grow and learn as new data is fed to it. Is this something you would be interested in?

**Comment by nabanitade** [comment]
Hey i think that would be interesting to work on.

**Comment by tbrianjones** [comment]
This would be really interesting to work on and is the underlying part of this project that really tackles the core issue here. Philosophically, How do we classify news, and what are the baseline assumptions or morals or biases that we start from?

**Comment by dchersey** [comment]
EXACTLY!  It seems to me there are 3 categories of classifiable news 

1. Things that can be discredited by multiple "neutral" sources
2. Things that can be authenticated by multiple "neutral" sources
3. Everything else :)

It's mostly number 3 that gives us problems ... but I would suggest that we don't necessarily need to be black & white about it.  A solution could be refined as
a) simply labeling it as 'debatable'
b) communicating some statistical info about agreement/disagreement in the community at large (need to define community - could be users, or could be balance of authoritative sources that agree/disagree, or both)
c) doing b, but providing a drill-down that exposes the biases, values, or backgrounds statistically of those who agree/disagree ... therefore communicating a holistic picture of the lenses through which we globally assess the truth of information and allowing the viewer to assign personal truth in that context. 

c is HARD. But something approaching it would be quite interesting to build....

**Comment by baileyfriend** [comment]
That being said, would you be ok with me creating a fork and attempting some of this? I am a student as well, so I can't promise any sort of timeframe.

**Comment by baileyfriend** [comment]
I think a big part of the question then is what do we define as neutral sources?

**Comment by dchersey** [comment]
Fine with me!  But I'm not part of the Princeton hack team :). That said, I would love to kick solutions to this around with you and help in any way I can.

**Comment by anantdgoel** [comment]
@baileyfriend Feel free to fork it, edit it and then send in a pull request. We love open source developers and could use all the help :)

**Comment by baileyfriend** [comment]
Fantastic! I will help whenever I have the open time, though right now is crunch time for final projects and exams. Hopefully I'll be able to contribute more in December once classes are over. Feel free to email me personally if you'd like to communicate any specific ideas of what you'd like to see. (freund.bailey@gmail.com)
